src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -17,8 +17,8 @@
 @Service
 @Transactional
 public class LineService {
-    private LineRepository lineRepository;
-    private StationService stationService;
+    final private LineRepository lineRepository;
+    final private StationService stationService;
```

Comment:
[구글 자바 컨벤션](https://google.github.io/styleguide/javaguide.html#s4.8.7-modifiers)에 따르면 접근 제어자 다음에 final 선언하는 것을 권장합니다.
```suggestion
    private final LineRepository lineRepository;
    private final StationService stationService;
```